andela/rainier-rc
# #151994136 Integrate payment with paystack

#### What does this PR do?
Add paystack payment method
#### Description of Task to be completed?
-Give users paystack payment option
#### How should this be manually tested?
-Add product to cart
-Click the checkout button
-On the checkout page, you'll see the paystack payment option button
-Click the button to make payment
#### What are the relevant pivotal tracker stories?
#151994136
#### Screenshots (if appropriate)
<img width="1432" alt="screen shot 2017-10-29 at 9 47 15 am" src="https://user-images.githubusercontent.com/8515311/32141996-8261b50a-bc8e-11e7-9c3d-a0c7449e2a42.png">
<img width="1430" alt="screen shot 2017-10-29 at 9 49 08 am" src="https://user-images.githubusercontent.com/8515311/32141997-8ba1e8e2-bc8e-11e7-9414-5696067e10e3.png">
<img width="1432" alt="screen shot 2017-10-29 at 9 49 28 am" src="https://user-images.githubusercontent.com/8515311/32141998-8bc4661a-bc8e-11e7-9d14-26a0d803c044.png">

#### Questions:
None
